docs/community/contribute/compliance.md

```diff
@@ -0,0 +1,151 @@
+
+For example, the following check is detailed in the AWS EKS CIS v1.4 Benchmark:
+`3.1.2 Ensure that the kubelet kubeconfig file ownership is set to root:root (Manual)`
+
+This check can be found in the general K8s CIS Compliance Benchmark: `k8s-cis-1.23.yaml` ([Link](https://github.com/aquasecurity/trivy-policies/blob/31e779916f3863dd74a28cee869ea24fdc4ca8c2/specs/compliance/k8s-cis-1.23.yaml#L480))
+
+Thus, we can use the information already present:
+
+```
+  - id: 3.1.2
+    name: Ensure that the kubelet service file ownership is set to root:root (Manual)
+    description: Ensure that the kubelet service file ownership is set to root:root
+    checks:
+      - id: AVD-KCV-0070
+    severity: HIGH
+```
+
+- The `ID`, `name`, and `description` is taken directly from the AWS EKS CIS Benchmarks
+- The `check` and `severity` are taken from the existing complaince check in the `k8s-cis-1.23.yaml`
+
+
+#### 2. Referencing a check manually that is not part of the Trivy default checks
+
+If the check does not already exist in the [Aqua Vulnerability Database](https://avd.aquasec.com/) (AVD) and is not part of the trivy-policies, the fields in the compliance spec for the check have to be populated manually. This is done by referencing the information in the official compliance specification.
+
+Below is the beginning of the information of the EKS CIS Benchmarks v1.4.0:
+
+![EKS Benchmarks 2.1.1](../../imgs/eks-benchmarks.png)
+
+The corresponding check in the `control` section will look like this:
+
+```
+  - id: 2.1.1
+    name: Enable audit Logs (Automated)
+    description: |
+      Control plane logs provide visibility into operation of the EKS Control plane components systems. 
+      The API server audit logs record all accepted and rejected requests in the cluster. 
+      When enabled via EKS configuration the control plane logs for a cluster are exported to a CloudWatch 
+      Log Group for persistence.
+    checks: null
+    severity: MEDIUM
+```
+
+- Again, the `id`, `name` and `description` are taken directly from the EKS CIS Benchmarks v1.4.0
+- The `checks` is in this case `null` as the check is not currently present in the AVD and does not have a check in the [trivy policies](https://github.com/aquasecurity/trivy-policies/tree/main/checks) repository
+- Since the check does not exist in Trivy, the `severity` will be `MEDIUM`. However, in some cases, the compliance report e.g. the CIS Benchmark report will specify the severity
+
+#### Contributing new checks to trivy-policies
+
+All of the checks in trivy-policies can be referenced in the compliance specs.
+To write new Rego checks for Trivy, please take a look at the contributing documentation for checks.
+
+### Test the Compliance Spec
+
+To test the compliance check written on a local Kubernetes cluster, pass the new path into the Trivy scan through the `--compliance` flag:
```

Comment:
compliance are not specific to k8s. fine to include an example just generalize the language please

Reply:
ah yes, of course